crypto/aes/asm/bsaes-armv8.pl

```diff
@@ -981,7 +1019,13 @@
 //   No output registers, usual AAPCS64 register preservation
 bsaes_cbc_encrypt:
         cmp     x2, #128
+#ifdef __APPLE__
```

Comment:
without this we get

`error: conditional branch requires assembler-local label. '_AES_cbc_encrypt' is external.`